src/modules/RA/DataLayer/components/FieldUpdater.js

```diff
@@ -1,13 +1,10 @@
 import { useEffect } from 'react';
-import get from 'lodash.get';
 /* eslint-disable import/no-extraneous-dependencies */
```

Comment:
`es-lint disable` is still accurate?

Reply:
I think so.

`react-final-form` is currently defined as root dependency, but I think it should be removed from package.json and be brought as `react-admin` sub-dependency.

Currently: 
```sh
$ npm ls react-final-form
terra-admin@2.14.1 ~/projets/Terralego/terra-admin
├─┬ react-admin@3.8.5
│ └── react-final-form@6.5.1 
└── react-final-form@6.5.1 
```

Reply:
As discussed with @jrmi, we'll try to define `react-final-form` *(and @material-ui/core)* as peerDepedency, to avoid having de setup so much linting exceptions.

Reply:
`react-final-form` is already a peerDep of react-admin so we require it as root dependency.  
Linting manual disabling is so not needed anymore.